src/Backends/SystemUpdate.vala

```diff
@@ -108,6 +108,13 @@ public class SettingsDaemon.Backends.SystemUpdate : Object {
                 0 //FIXME: Is there a way to get update size from PackageKit
             };
 
+            update_state (AVAILABLE);
+
+            if (settings.get_boolean ("automatic-updates")) {
```

Comment:
Should this also be `!force`? If the user triggers a refresh manually does that mean we should let them install manually too?

Reply:
Yeah I think that'd be better